protocol/scripts/genesis/Dockerfile

```diff
@@ -0,0 +1,56 @@
+# NB: This is a digest for a multi-arch manifest list, you will want to get this by running
```

Comment:
What's the difference between this dockerfile and `protocol/Dockerfile`? Can we avoid the duplication and extra maintenance by reusing it like we do in the staging dockerfile, localnet dockerfile, etc?

If the only difference is installing jq and dasel, it's OK to add it to the original docker image IMO.

Reply:
Updated to use the `dydxprotoocl-base` image as base